spec/helpers/feedbacks_helper_spec.rb

```diff
@@ -46,6 +46,17 @@ class Klass
         klass.open_requests(user).should == 5
       end
     end
+
+    describe 'unsubmitted external feedback' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.